internal/application/application.go

```diff
@@ -51,6 +51,13 @@ func (a *Application) Delete(gitea GiteaInterface) error {
 		return pkgerrors.Wrap(err, "failed to delete application deployment")
 	}
 
+	err = a.kubeClient.Kubectl.ExtensionsV1beta1().Ingresses(a.Organization).
```

Comment:
Does this mean we need to delete the `Service` explicitly too?

Reply:
Possibly yes. I will investigate.

Reply:
Defered to separate ticket.